ui/src/dashboards/components/TemplateVariableRow.js

```diff
@@ -30,7 +32,9 @@ const RowValues = ({
   onStartEdit,
   autoFocusTarget,
 }) => {
-  const _values = values.map(({value}) => value).join(', ')
+  const _values = uniq(values.map(({value}) => value))
```

Comment:
Interesting use case... I could switch to set, but this data is otherwise expected to be an array throughout the app. Think it's worth the refactor?